docs/contributing/standards.rst

```diff
@@ -122,6 +122,12 @@ Example
         umVEX-8 series artifacts will need to be updated to umVX format
         with the consortium or vendor toolset.
 
+.. note::
+
+  You can install `this commit-msg script`_ to automatically check the that you
+  follow these guidelines when you commit.
+
+.. _`this commit-msg script`: https://gist.github.com/Joshua-Anderson/36e8bcdbcb17407215ba
```

Comment:
Should this script be committed as part of this PR instead of left in a gist? It could live in /contrib/util or in /tests/bin.